dlib/dnn/loss.h

```diff
@@ -3641,7 +3641,7 @@ namespace dlib
                                                              h / (1 - h) * anchors[a].height));
                                 for (long k = 0; k < num_classes; ++k)
                                 {
-                                    const float conf = out_data[tensor_index(output_tensor, n, a * num_feats + 5 + k, r, c)] * obj;
+                                    const float conf = obj * out_data[tensor_index(output_tensor, n, a * num_feats + 5 + k, r, c)];
```

Comment:
Putting `obj` at the beginning makes it easier to read, otherwise it might be overlooked, since the line is relatively long.